app/components/admin/statements/adjustments.html.erb

```diff
@@ -0,0 +1,28 @@
+<%=
+  govuk_summary_card(title: "Additional adjustments", html_attributes: { id: "adjustments" }) do |card|
+    if adjustment_editable?
+      card.with_action { govuk_link_to("Add", new_admin_finance_statement_adjustment_path(statement), visually_hidden_suffix: "adjustment") }
+    end
+
+    card.with_summary_list do |sl|
+      adjustments.each do |adjustment|
+        sl.with_row do |row|
+          row.with_key(text: adjustment.payment_type)
+          row.with_value(text: number_to_pounds(adjustment.amount))
+
+          if adjustment_editable?
+            row.with_action(text: "Change", href: edit_admin_finance_statement_adjustment_path(statement, adjustment), visually_hidden_text: "adjustment")
+            row.with_action(text: "Remove", href: delete_admin_finance_statement_adjustment_path(statement, adjustment), visually_hidden_text: "adjustment")
+          end
+        end
+      end
+
+      if adjustments.any?
```

Comment:
When there are no adjustments the box looks a bit awkward.

![image](https://github.com/user-attachments/assets/a962b665-ce92-4753-9097-fc8aaf2c7322)

Maybe a line of text would make it clearer?

![Screenshot From 2025-06-16 12-53-35](https://github.com/user-attachments/assets/bcbc7b49-fa33-4515-b2a3-5b4331529120)

Reply:
Added empty view